projects/zilliqa-duel/project.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Zilliqa Duel
+description: False
+categories: false
```

Comment:
@AmritKumar should we include in those projects which have no information whatsoever? `false` for categories will currently break the script. I won't update the script at present unless this is a case that might happen for future projects?

Reply:
We have fixed the issue.

Reply:
@AmritKumar the current categories still show false though

Reply:
```suggestion
categories: dapp,gaming
```